docs/en/build-store/refresh-control.md

```diff
@@ -0,0 +1,240 @@
+
+This is especially useful for device manufacturers to mediate releases of the *core* snap, allowing time for testing and validation of new revisions on their device models and compatibility with their software.  Note this is separate from the ability for device managers to control the schedule of refreshes and revision control via a snap proxy.
+
+
+## Preliminary setup
+
+Establishing the relationship between a gating and gated snap is required only once and can only be performed by a user with reviewer permission for the gating snap's store.
+
+Ask a store administrator/reviewer to edit the package declaration for the gating snap on the [snap's page][1]. The administrator should:
+
+* Click on "review capabilities"
+* Under "refresh control" put a json list with the snap IDs of the snaps you want gated: \["xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx", ...\]
+* The list can, of course, contain more than one snap ID, in which case this gating snap will be able to control updates for all the gated snaps.
+
+## Publisher setup
+
+Since gating involves pushing [`validation` assertions][2] which are signed by the publisher indicating their authorization to restrict updates of snaps, some GPG keys must be generated and published to the store. If a store key already exists, it can be used. If not, one must be generated and registered.
+
+1. `snapcraft create-key` to create a key. (by default it's named "default" but a key name can be given to create a custom one so e.g. each snap is signed with its own key). Do not forget the passphrase used to secure the key or misplace the key files.
+
+    ```
+    $ snapcraft create-key my-key
+    Passphrase:
+    Confirm passphrase:
+    $ snapcraft list-keys
+        Name          SHA3-384 fingerprint
+    -   my-key       mfomTtPB1cE3IFs51NtdnFoPlQwJxFgxOMU_q0mPkH7M2gKB-4m28d99XrVjA53B  (not registered)
+    ```
+
+2. `snapcraft register-key <the-key-name>` to register it with the assertion server.
+
+    ```
+    $ snapcraft register-key my-key
+    Enter your Ubuntu One e-mail address and password.
+    If you do not have an Ubuntu One account, you can create one at https://dashboard.snapcraft.io/openid/login
+    Email: roadmr.developer@example.com
+    Password:
+    Second-factor auth: 111111
+    Registering key ...
+    Done. The key "my-key" (mfomTtPB1cE3IFs51NtdnFoPlQwJxFgxOMU_q0mPkH7M2gKB-4m28d99XrVjA53B) may be used to sign your assertions.
+    ```
+
+## Publishing workflow, examples
+
+At this point, the developer can issue validations asserting which revision of the gated snap is allowed to be installed when the gating snap is installed on a device. For example, specify that if the roadmr-gating snap is installed, the roadmr-gated snap can only be installed or refreshed to revision 2.
```

Comment:
not "developer" I think. Is it not the gating snap's Publisher account?

Reply:
Indeed, changed the term.